cmd/apiserver-boot/boot/init_repo/repo.go

```diff
@@ -106,7 +106,7 @@ func createBazelWorkspace() {
 	}
 	path := filepath.Join(dir, "WORKSPACE")
 	util.WriteIfNotFound(path, "bazel-workspace-template", workspaceTemplate, nil)
-	path = filepath.Join(dir, "BUILD.bazel")
+	path = filepath.Join(dir, "BUILD")
```

Comment:
I think this should not be changed. `BUILD.bazel` is better for new bazel project

Reply:
others lgtm

Reply:
oh misread, you prefer `BUILD.bazel` over `BUILD`